### PR TITLE
Bugfix/platfowner/bugfix

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Fix types and comments

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1252

### CODE REVIEW

File: src/provider.ts [hunk 1]
```diff
@@ -39,9 +39,9 @@ export default class Provider {
 
   /**
    * Fetches the blockchain node's address.
-   * @returns {Promise<number>} The address value.
+   * @returns {Promise<any>} The return value of the blockchain API.
```
Comment (jiyoung-an) [inline src/provider.ts:42]: is it a string type?

**Review by jiyoung-an — APPROVED** [review]
LGTM with a question. thanks!

File: src/provider.ts (hunk 1 above)
Comment (platfowner) [reply src/provider.ts]: right. just fixed, thanks!

**Comment by platfowner** [comment]
Thanks for the review! @jiyoung-an Now merging..

**Pull request merged**